hi.js

```diff
@@ -1006,8 +1006,8 @@ server.message.kicked {0} को किक किया गया
 server.message.banned {0} को बैन किया गया
 server.message.votekicked {0} was vote kicked
 server.message.arrested {0} को K.P.D ने गिरफ्तार कर लिया
-event.easter.golden.spawn A Golden egg is near
-event.easter.golden.despawn A Golden egg has gone
+event.easter.golden.spawn A गोल्डन एग is near
+event.easter.golden.despawn A गोल्डन एग has gone
```

Comment:
same thing, no need of mixed statements

Reply:
just leave this in English it's partially translated.